app/services/highlighter/index.ts

```diff
@@ -706,17 +944,53 @@ export class HighlighterService extends StatefulService<IHighligherState> {
       return;
     }
 
-    let clips = this.views.clips
-      .filter(c => c.enabled)
-      .map(c => {
-        const clip = this.clips[c.path];
+    let clips: Clip[] = [];
+    if (streamId) {
+      clips = this.getClips(this.views.clips, streamId)
+        .filter(clip => clip.enabled)
+        .filter(clip => {
```

Comment:
not sure why you're filtering twice here, could just add `clip.enabled` to the below check right?